src/main/scala/io/iohk/ethereum/network/PeerManagerActor.scala

```diff
@@ -48,6 +47,9 @@ class PeerManagerActor(
     case Terminated(ref) =>
       peers -= ref.path.name
 
+    case broadcast: PeerActor.Broadcast[_] =>
+      peers.values.foreach(_.ref ! broadcast)
```

Comment:
we can just send `SendMessage(...)` to peers here. That way we don't have to handle `Broadcast` in `PeerActor`.